app/components/baseComponents/components/inputs/text/text.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import { useState, KeyboardEvent } from "react";
+import styles from "./text.module.scss";
+
+const defaultValidationPattern = "^[a-zA-Z()_0-9]+$";
+
+export default function TextInput({
+    label,
+    placeholder,
+    isRequired=false,
+    validationPattern=defaultValidationPattern,
+    id
+}:{
+    label: string,
+    placeholder: string,
+    isRequired: boolean,
+    validationPattern?: string,
```

Comment:
Does this still need to be nullable if we provide a default value?

Reply:
Technically no